app/data/test_navigation.json

```diff
@@ -0,0 +1,667 @@
+{
```

Comment:
Of course we need to test it fully but does that mean having a home insurance questionnaire. What does it mean in this case by testing it fully i.e. Do we only need a repeating group and a non repeating group? Should these be separate schemas for clarity?